backend/src/main/resources/db/migration/V1_1_1__update-mark.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE mark_value
+  ADD COLUMN description VARCHAR(255) NOT NULL;
```

Comment:
Just fyi this wouldn't work if there were entries in this table. You'd have to add a nullable column first, set all columns with null to a default value and then make the column nullable.